packages/shared-internal/src/clients/novu-client.ts

```diff
@@ -0,0 +1,12 @@
+import { createStepSchemaClient } from './step-schemas-client';
```

Comment:
This belongs to the Node SDK package. It's a good opportunity to make touch-ups there and enhance it with the HTTP client of the new environment and workflow endpoints.

It's perfectly OK if @novu/node exports two types of clients for v1 and V2 endpoints,.

Reply:
you don't want to expose it to customers, it's internal

Reply:
removed it from shared, will keep it in api for e2e, will move it after this merges